resources/templates/ha-vpn-interop-guide-template/index.md

```diff
@@ -0,0 +1,836 @@
+the table describes how to set Linux environment variables to hold the values
+you need for these parameters.
+
+<table>
+<thead>
+<tr>
+<th><strong>Parameter description</strong></th>
+<th><strong>Placeholder</strong></th>
+<th><strong>Example value</strong></th>
+</tr>
+</thead>
+<tbody>
+
+<tr>
+<td>Vendor name</td>
+<td><code>[VENDOR_NAME]<code></td>
+<td>(Your product's vendor name. This value should have no spaces or
+punctuation in it other than underscores or hyphens, because it will be
+used as part of the names for GCP entities.)</td>
+</tr>
+
+<tr>
+<td>GCP project name </td>
+<td><code>[PROJECT_NAME]<code></td>
+<td><code>vpn-guide<code></td>
+</tr>
+
+<tr>
+<td>Shared secret</td>
+<td><code>[SHARED_SECRET]<code></td>
+<td>See <a
+href="https://cloud.google.com/vpn/docs/how-to/generating-pre-shared-key">Generating
+a Strong Pre-shared Key</a>.</td>
+</tr>
+
+<tr>
+<td>VPC network name</td>
+<td><code>[NETWORK]<code></td>
+<td><code>network-a<code></td>
+</tr>
+
+<tr>
+<td>Subnet mode</td>
+<td><code>[SUBNET_MODE]<code></td>
+<td><code>custom<code></td>
+</tr>
+
+<tr>
+<td>BGP routing mode</td>
+<td><code>[BGP_ROUTING_MODE]<code></td>
+<td><code>global</td>
+</tr>
+
+<tr>
+<td>Subnet on the GCP VPC network (for example, <code>vpn-vendor-test-network</code>)</td>
+<td><code>[SUBNET_NAME_1]<code></td>
+<td><code>subnet-a-central<code></td>
+</tr>
+    
+<tr>
+<td>Subnet on the GCP VPC network (for example, <code>vpn-vendor-test-network</code>)</td>
+<td><code>[SUBNET_NAME_2]<code></td>
+<td><code>subnet-a-west<code></td>
+</tr>
+    
+<tr>
+<td>GCP region. Can be any region, but it should be geographically close to the
+on-premises gateway.</td>
+<td><code>[REGION1]<code></td>
+<td><code>us-central1<code></td>
+</tr>
+    
+<tr>
+<td>GCP region. Can be any region, but it should be geographically close to the
+on-premises gateway.</td>
+<td><code>[REGION2]<code></td>
+<td><code>us-west1<code></td>
+</tr>
+
+<tr>
+<td>IP address range for the GCP VPC subnet (<code>vpn-subnet-1</code>)</td>
+<td><code>[RANGE_1]<code></td>
+<td><code>10.0.1.0/24<code></td>
+</tr>
+    
+<tr>
+<td>IP address range for the GCP VPC subnet (<code>vpn-subnet-2</code>)</td>
+<td><code>[RANGE_2]<code></td>
+<td><code>10.0.2.0/24<code></td>
+</tr>
+
+<tr>
+<td>IP address range for the on-premises subnet. You will use this range when
+creating rules for inbound traffic to GCP.</td>
+<td><code>[IP_ON_PREM_SUBNET]<code></td>
+<td><code>192.168.1.0/24<code></td>
+</tr>
+
+<tr>
+<td>External static IP address for the first internet interface of <vendor
+name><product-name></td>
+<td><code>[ON_PREM_GW_IP_0]</code> </td>
+<td>For example, <code>199.203.248.181</code></td>
+</tr>
+
+<tr>
+<td>External static IP address for the second internet interface of <vendor
+name><product-name></td>
+<td><code>[ON_PREM_GW_IP_1]</code> </td>
+<td>For example, <code>199.203.248.182</code></td>
+</tr>
+
+<tr>
+<td>HA VPN Gateway</td>
```

Comment:
I might split this into a second table and label it VPN gateway information.
You're also missing the autogenerated external IPs for each interface of the HA VPN gateway.